zproject/dev_settings.py

```diff
@@ -41,3 +41,4 @@
 
 # Enable inline open graph preview in development for now
 INLINE_URL_EMBED_PREVIEW = True
+FIELDS_STORED_IN_SESSION = ['subdomain']
```

Comment:
Shouldn't this be `SOCIAL_AUTH_FIELDS_STORED_IN_SESSION` and be set in DEFAULT_SETTINGS so it applies to production as well?